scripts/ci-integration.sh

```diff
@@ -19,7 +19,7 @@ set -o nounset
 set -o pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-# shellcheck source=../hack/ensure-go.sh
+# shellcheck source=/dev/null
```

Comment:
This causes SC1091 https://github.com/koalaman/shellcheck/wiki/SC1091
IMO, it is OK to ignore this as we run `shellcheck` on all `.sh` files in the repo.
Also, the correct way to ignore this seems to be like
```
# shellcheck disable=SC1091
```
However, the `/dev/null` approach seems like the prefered way in the repo.

LMK if the way I've understood this is not correct.

Reply:
Reading that, I'm wondering if we should be invoking `shellcheck` with the `-x` flag to allow it to follow the file.

Reply:
I tried that, but it didn't work.

Reply:
I was able to get that to work. The last time I tried the `-x` I wasn't providing the path to the sourced file correctly.

Reply:
I've fixed it now.